src/_settings.scss

```diff
@@ -4,6 +4,8 @@
 // library namespace
 $fd-namespace: fd !default;
 
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/sass_variables.scss";
```

Comment:
We don't need the scss variables. The foundation should be in place to simply use the css variables.

Reply:
Removed the import